### DISCUSSION TITLE
How to add base-400, or another variants to DaisyUI?

### DISCUSSION THREAD

**Discussion by hshoob** [saadeghi/daisyui#907] [Q&A] [answered]
Is there a way to extend the DaisyUI theme by adding custom variants shown in the image below? Also, is there a way to implement a shading system in DaisyUI, e.g., 100 to 900 or 100 to 500 shades of primary/secondary/success, etc.? So DaisyUI can work as a complete theming solution. Thank you. 

<img width="545" alt="Screen Shot 2022-06-28 at 12 02 37 AM" src="https://user-images.githubusercontent.com/18516326/176026040-59ee3f94-0207-4233-a7de-24b8c658f541.png">

**Comment by saadeghi** [comment, marked as answer]
See this:  
https://github.com/saadeghi/daisyui/discussions/653#discussioncomment-2438428

**Reply by hshoob** [reply]
Thank you. DaisyUI is incredible! I wish you had the time to integrate Tailwind Headless UI (https://headlessui.com/) into DaisyUI.

**Reply by saadeghi** [reply]
You can already use them together and it's super easy. It doesn't really need integration. One handles the functionality and the other one handles the design decisions.
